dipy/tracking/local/tests/test_tracking.py

```diff
@@ -567,7 +567,9 @@ def test_bootstap_peak_tracker():
     seeds = [np.array([0., 1., 0.]), np.array([2., 4., 0.])]
 
     tc = BinaryTissueClassifier((simple_image > 0).astype(float))
-    boot_dg = BootDirectionGetter.from_data(data, csd_model, 60)
+    sphere = HemiSphere.from_sphere(get_sphere('symmetric724'))
```

Comment:
Here as well: why are we still using symmetric?

Reply:
almost the same comment as above. we can create an issue to spend more time on these 2 specifics tests and update them